docker-compose.yml

```diff
@@ -1,13 +1,10 @@
-version: "2.1"
+version: "3.7"
 
 services:
   oereb-server:
     build:
       context: .
       dockerfile: Dockerfile
-    depends_on:
-      oereb-db:
-        condition: service_healthy
```

Comment:
Is there a particular reason for removing the health check?
This is an issue for me because when `oereb-server` first tries to connect with `oereb-db`, the DB is not up yet -> the server crashes with a Python error

Reply:
I tried to use a newer docker-compose version (3.7) to be able to support the `user` setting to sync in container user with host user. This way we wouldn't have file generated on host mount as root. But this is  [WIP]

Reply:
@vvmruder  you can change the command of the container to use `exec` with a wait condition that checks if the database is available with a timeout. Something like 
```bash
exec wait-for -t 120 <some-health-check-code-that-points-to-postgres> -- <your-usual-startup-command-here>
```
 as workaround. Otherwise it should be the main application (pyramid_oereb wsgi) that should wait until the DB is up at startup with a timeout as well.

Reply:
Thats true. I will see if I find a simple solution to empower pyramid_oereb to check when DB is available.

Reply:
I did an update. Please have a look.

Reply:
I checked your code, it seems good to me, thanks